### PR TITLE
[WIP] Helm3

### PR DESCRIPTION
**Pull request by msenmurugan**
- [x] Vendor Helm
- [x] helm install
- [ ] helm create releases
- [ ] helm upgrade
- [ ] helm delete

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,49 +3,55 @@ module github.com/blackducksoftware/synopsys-operator
...
-	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/klog v0.4.0 // indirect
-	k8s.io/kube-openapi v0.0.0-20190722073852-5e22f3d471e6 // indirect
-	k8s.io/utils v0.0.0-20190809000727-6c36bc71fc4a // indirect
+	helm.sh/helm/v3 v3.1.0
+	k8s.io/api v0.17.3
+	k8s.io/apiextensions-apiserver v0.17.2
+	k8s.io/apimachinery v0.17.3
+	k8s.io/cli-runtime v0.17.2
+	k8s.io/client-go v0.17.2
```
Comment (yashbhutwala) [inline go.mod]: k8s is so much cleaner now 😍

File: pkg/util/helm_helpers.go [hunk 2]
```diff
@@ -27,27 +27,138 @@ import (
...
+	}
+
+	validInstallableChart, err := isChartInstallable(chartRequested)
+	if !validInstallableChart {
+		return nil, err
+	}
+
+	if chartRequested.Metadata.Deprecated {
+		log.Info("WARNING: This chart is deprecated")
+	}
```
Comment (yashbhutwala) [inline pkg/util/helm_helpers.go]: That's not bad at all to vendor 😄

**Comment by msenmurugan** [comment]
merged with Michael Hammers PR

**Pull request closed without merging**